src/telcameras_v2/management/commands/parse_ingress.py

```diff
@@ -0,0 +1,36 @@
+import json
+
+from django.core.management.base import BaseCommand
+
+from ingress.parser import IngressParser
+from telcameras_v2.data_conversions import data_to_observation
+from telcameras_v2.serializers import ObservationSerializer
+
+
+class TelcameraParser(IngressParser):
```

Comment:
Deze class zou ik liever in een aparte file neerzetten net als bij de IngressParser gedaan is (bijvoorbeeld `telcameras_v2/ingress_parser.py`)